src/pages/Detail/Detail.js

```diff
@@ -1,8 +1,161 @@
 import React, { Component } from 'react';
+import Nav from '../../components/Nav/Nav';
+import TCLogin from '../../components/TestComp/TCLogin';
+import './Detail.scss';
 
 export class Detail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      quantity: 1,
+      loginModal: false,
+    };
+  }
+
+  handleInput = e => {
+    const re = /^[0-9\b]+$/;
+    const { value } = e.target;
+
+    if (re.test(value)) {
+      this.setState({ quantity: Number(value) });
+    }
+    if (value === '') {
```

Comment:
```suggestion
    if (!value) {
```
falsy 값 특성 이용해서 위와 같이 표현할 수 있습니다